core-bundle/src/Resources/views/Frontend/preview_toolbar_base_js.html.twig

```diff
@@ -253,28 +255,31 @@
 
             if (document.createElement('datalist') && window.HTMLDataListElement) {
                 const userField = form.querySelector('input[name="user"]');
-                const userList = form.querySelector('datalist[id="userlist"]');
 
-                userField.addEventListener('keyup', function () {
-                    if (userField.value.length < 2) {
-                        return;
-                    }
-
-                    const body = {
-                        FORM_SUBMIT: 'datalist_members',
-                        REQUEST_TOKEN: form.querySelector('input[name="REQUEST_TOKEN"]').value,
-                        value: userField.value
-                    };
-
-                    request('POST', form.action, new URLSearchParams(body), function () {
-                        userList.innerHTML = '';
-                        JSON.parse(this.response).forEach(item => {
-                            const option = document.createElement('option');
-                            option.value = item;
-                            userList.appendChild(option);
-                        });
-                    }, false);
-                });
+                if (userField) {
+                    userField.addEventListener('keyup', function () {
```

Comment:
On a side note unrelated to this PR: We probably want to use the [*input* event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event) instead. Afaik *keyup* isn't triggered when pasting for instance.

Reply:
See #2079